setup/provinces/te_default.txt

```diff
@@ -87,16 +98,16 @@
 	trade_goods=""
 }
 #
-5971=

```

Comment:
If you're leaving commented out blocks, I suggest adding a comment with an explanation here. Maybe copy the explanation from the changelog.